lib/utils/logger/index.js

```diff
@@ -302,6 +303,103 @@ class Logger {
     console[type](message);
   }
 
+  getAssertionErrorContent(err, modulepath) {
+    const content = [];
+    let stack = err.stack || err.stackTrace;
+    content.push(`  ${colors.light_red(`→ ${err.name}`)}`);
+
+    if (err.fullMsg || err.message){
+      content.push(`\n    ${colors.red(err.message)}`);
+      if (err.detailedErr) {
+        content.push(`\n    ${colors.light_green(err.detailedErr)}`);
+        if (err.extraDetail) {
+          content.push(`\n    ${colors.light_green(err.extraDetail)}`);
+        }
+      }
+    }
+
+    if (stack) {
+      stack = '\tat' + stack.split(/ {4}at/g).slice(1).join('\tat');
+      content.push(colors.brown('\n    Error location :'));
+      content.push(beautifyStackTrace(stack, true, modulepath));
+    
+      content.push(colors.brown('    Stack Trace :'));
+      content.push(colors.stack_trace(`${stack}\n`));
+    }
+
+    return content.join('\n');
+  }
+
+  getFailedAssertions(assertions, modulepath) {
+    return assertions.reduce((prev, a) => {
+      if (a.failure !== false) {
+        prev.push(this.getAssertionErrorContent(a, modulepath));
+      }
+
+      return prev;
+    }, []).join('\n');
+  }
+
+  getErrorContent(error, modulepath) {
```

Comment:
This doing almost the same thing as `getAssertionErrorContent`, can't these be merged into a single function?

Reply:
Fixed in #[727f862](https://github.com/nightwatchjs/nightwatch/pull/3233/commits/727f86213aa867e7ecc2b2175d9d83a7d558d527)